src/sentry/notifications/manager.py

```diff
@@ -306,19 +306,6 @@ def _filter(
 
         return self.filter(query)
 
-    def remove_for_user(self, user: User, type: NotificationSettingTypes | None = None) -> None:
```

Comment:
@corps I guess these two functions are never used?

Reply:
Afaik they were only ever used in tests.

Reply:
@corps is there a reason we never did?

Reply:
Non philosophical anyways, I just think likely the methods were made to make the tests easier to write.